contracts/levels/AlienCodex.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.4.24;
+
+import 'zeppelin-solidity/contracts/ownership/Ownable.sol';
+
+contract AlienCodex is Ownable {
+
+  bool public contact;
+  bytes32[] public codex;
+
+  modifier contacted() {
+    assert(contact);
+    _;
+  }
+  
+  function make_contact(bytes32[] _impossible_message) public {
```

Comment:
Seems as though `_impossible_message` gives it away a touch - what if you named it something in keeping with the **_awesome_** theme, like `_firstContactMessage`, or you could emphasize that the message has to be big and important with something like `_momentousMessage`?

Reply:
_firstContactMessage is much better. Changed - thanks!